assignments/sa/websockets/index.md

```diff
@@ -18,7 +18,7 @@ We're going to build a [websocket](https://en.wikipedia.org/wiki/WebSocket) base
 🚀 Clone your **Lab3 with firebase**.  Create a new branch with `git checkout -b websockets`. We'll need to refactor this very slightly to not use firebase but use our new backend instead, and using a branch seems reasonable for this.
 
 
-🚀 Clone or Fork [express-babel-starter](https://github.com/dartmouth-cs52-19s/express-babel-starter) for our new server component. You can create a new github repo in your own account if you want for this and add it as a remote. 
+🚀 Clone or Fork [express-babel-starter](https://github.com/dartmouth-cs52/express-babel-starter) for our new server component. You can create a new github repo in your own account if you want for this and add it as a remote. 
```

Comment:
when i made the server, i found that the most updated version existed at this link

Reply:
perfect yup - i used to fork them over but then just settled on one